app/presenters/menu/default_menu.rb

```diff
@@ -89,9 +89,10 @@ def infrastructure_menu_section
       def physical_infrastructure_menu_section
         Menu::Section.new(:phy, N_("Physical Infrastructure"), 'fa fa-plus fa-2x', [
           Menu::Item.new('ems_physical_infra',    N_('Providers'), 'ems_physical_infra',    {:feature => 'ems_physical_infra_show_list'},    '/ems_physical_infra'),
+          Menu::Item.new('physical_chassis', N_('Chassis'), 'physical_chassis', {:feature => 'physical_chassis_show_list'}, '/physical_chassis'),
           Menu::Item.new('physical_rack', N_('Racks'), 'physical_rack', {:feature => 'physical_rack_show_list'}, '/physical_rack'),
-          Menu::Item.new('physical_switch', N_('Switches'),  'physical_switch', {:feature => 'physical_switch_show_list'}, '/physical_switch'),
           Menu::Item.new('physical_server', N_('Servers'),   'physical_server', {:feature => 'physical_server_show_list'}, '/physical_server'),
+          Menu::Item.new('physical_switch', N_('Switches'),  'physical_switch', {:feature => 'physical_switch_show_list'}, '/physical_switch'),
```

Comment:
Here I need a check if this works correctly in the RBAC form. I mean you have `physical_switch` and then check for `physical_switch_show_list`. I don't remember how exactly it work.

If you can check it and add a screenshot of the RBAC tree under roles. It would be great.

Reply:
@martinpovolny I only changed the order of the Physical Switch entry on the Menu. However, I am trying to add the entry for Physical Chassis.

See here the `physical_chassis_show_list` is under `physical_chassis_view`:
https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L6007
![show_list](https://user-images.githubusercontent.com/14334253/42291757-1b75f796-7fa4-11e8-9a13-4b868270e7fb.png)

And in the `miq_user_roles` the role for `physical_chassis_view` is declared for the users:
https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_user_roles.yml#L399

Thus, the features 'inside' the `physical_chassis_view` work. Right?